server/bin/pbench-server-put-shim.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+# This script is used to send the tarballs that a version 002 client
+# submits for further processing. It forwards the tarballs and their MD5
+# sums to the remote server using the server's HTTP PUT method
```

Comment:
Should we emphasize that "the remote server" is a 0.71 server? (0.72?, "user-authenticated", ... well, OK, I can't figure out the right terminology here, either, and maybe it's sufficiently obvious we must be talking about a "Pbench server" supporting a `PUT` operation...)

Reply:
Sure, I will change that